lib/stripe.rb

```diff
@@ -48,47 +50,41 @@
 module Stripe
   DEFAULT_CA_BUNDLE_PATH = __dir__ + "/data/ca-certificates.crt"
 
-  @app_info = nil
-
-  @api_base = "https://api.stripe.com"
-  @connect_base = "https://connect.stripe.com"
-  @uploads_base = "https://files.stripe.com"
-
-  @log_level = nil
-  @logger = nil
-
-  @proxy = nil
-
-  @max_network_retries = 0
-  @max_network_retry_delay = 2
-  @initial_network_retry_delay = 0.5
-
-  @ca_bundle_path = DEFAULT_CA_BUNDLE_PATH
-  @ca_store = nil
-  @verify_ssl_certs = true
+  # map to the same values as the standard library's logger
+  LEVEL_DEBUG = Logger::DEBUG
+  LEVEL_ERROR = Logger::ERROR
+  LEVEL_INFO = Logger::INFO
```

Comment:
Left the CA and LEVEL constants here in case folks reference them from outside the gem.

Reply:
Good call.